sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizeEntityAsyncClient.java

```diff
@@ -178,18 +180,24 @@
      * {@link RecognizeEntitiesResultCollection} from a {@link SimpleResponse} of {@link EntitiesResult}.
      *
      * @param documents The list of documents to recognize entities for.
-     * @param options The {@link TextAnalyticsRequestOptions} request options.
+     * @param options The {@link RecognizeEntitiesOptions} request options.
      * @param context Additional context that is passed through the Http pipeline during the service call.
      *
      * @return A mono {@link Response} that contains {@link RecognizeEntitiesResultCollection}.
      */
     private Mono<Response<RecognizeEntitiesResultCollection>> getRecognizedEntitiesResponse(
-        Iterable<TextDocumentInput> documents, TextAnalyticsRequestOptions options, Context context) {
+        Iterable<TextDocumentInput> documents, RecognizeEntitiesOptions options, Context context) {
+        String modelVersion = null;
+        boolean includeStatistics = false;
+        StringIndexType stringIndexType = StringIndexType.UTF16CODE_UNIT;
```

Comment:
because we use this everywhere, is there a way this could be a global constant in the library?
like `defaultStringInxedType` or something like that

Reply:
StringIndexType.UTF16CODE_UNIT is already a static object. It is a global constant already.